mlvm/test_mlvm.py

```diff
@@ -147,7 +149,8 @@ def test_mlvm_gpu(self, configuration, dask_runtime, rapids_runtime):
         master_accelerator="type=nvidia-tesla-t4",
         worker_accelerator="type=nvidia-tesla-t4",
         timeout_in_minutes=60,
-        metadata=metadata)
+        metadata=metadata,
+        boot_disk_size="500GB")
```

Comment:
it shouldn't be that much.  Maybe 80GB or 100GB if you want to be excessively careful.  Don't waste half a TB of pd-ssd on this, please.

Reply:
Ack.